composer.json

```
@@ -179,7 +179,7 @@
             "contao/newsletter-bundle": "Contao\\NewsletterBundle\\ContaoManager\\Plugin"
         },
         "symfony": {
-            "require": "4.4.*"
+            "require": "4.4.* || 5.2.*"
```

```suggestion
            "require": "4.4.* || ^5.2"
```

This will also allow Symfony packages that are no direct requirements in version 5.3.